src/sources/aws_ecs_metrics/mod.rs

```diff
@@ -142,43 +143,57 @@ async fn aws_ecs_metrics(
                             end: Instant::now()
                         });
 
-                        let byte_size = body.len();
```

Comment:
Tiny nit: You could leave this hear, as you do the same thing inside the `match` below.

Reply:
fixed 😉